crates/ibc/src/storage.rs

```diff
@@ -500,7 +500,8 @@ pub fn ibc_trace_key(
     addr: impl AsRef<str>,
     token_hash: impl AsRef<str>,
 ) -> Key {
-    ibc_trace_key_prefix(Some(addr.as_ref().to_string()))
+    // Remove '/' because IBC denom could have '/'
+    ibc_trace_key_prefix(Some(addr.as_ref().replace('/', "")))
```

Comment:
don't we want to preserve the entire trace? maybe we should replace `/` with some other character that is guaranteed to not appear in ICS-20 trace paths (`\x00`, for instance). when recovering the trace path from storage, we would replace `\x00` with `/` again, perhaps.

Reply:
ah but wait, how come `addr` has a slash at all? isn't this the account submitting tokens on the counterparty? for instance, `osmo1...` from osmosis

Reply:
Thanks, that's a good question.
The `addr` as a token address was for scanning all of the related tokens, but it isn't used anymore because we need to specify the token when querying balances.
(For example, `namadac balance --token uatom` showed balances of `uatom` related IBC tokens like `transfer/channel-12/uatom` and `transfer/channel-23/uatom` and so on.)
We don't have to store IBC traces for each token now. So, I removed these stores. (Existing traces are left on the current chain, but they're not used.)